docs/src/references/mappers/c8y-mapper.md

```diff
@@ -568,6 +568,247 @@ c8y/alarm/alarms/create
 
 </div>
 
+### Twin
+
+The `twin` metadata is mapped to [inventory data](https://cumulocity.com/guides/concepts/domain-model/#inventory) in Cumulocity.
```

Comment:
Two key points are missing:

- support for static fragments with the `inventory.json` file.
- automatic support for `c8y_Agent` and `type` fragments.